src/components/Navbar/interfaces.tsx

```diff
@@ -1,12 +1,14 @@
 export interface Brand {
   /** Clinic/Hospital name */
   label: string
+  /** Label color */
+  color?: string
   /** Determina the href */
   href?: string
   /** A path which contain the company icon/image */
   src?: string
   /** A click handle which will redirect the user to the respectivel webpage/path */
-  onClick: (event: React.MouseEvent<any>) => void
+  onClick?: (event: React.MouseEvent<any>) => void
```

Comment:
I think that this click handler could stay required. What do you think?

Reply:
I agree! At the moment I was thinking of people who plan on using for identification and aesthetic purposes or use another link to redirect to their homepage or about me but I definitely agree it should have an onClick functionality. 
What I would love to help with error handling but am not sure the best practice for this and where this would be done. I would like to be able to give the developer an error message if he tries to use the component without an onClick or with an href explaining the proper use of the properties. 
If anyone has a file or link with some best practices on error handling they would like me to follow I think internationalization for error handling will be very beneficial to the developers!